Lib/test/libregrtest/main.py

```diff
@@ -78,6 +78,14 @@ def __init__(self, ns: Namespace):
             self.ignore_tests: FilterTuple = tuple(ns.ignore_tests)
         else:
             self.ignore_tests = None
+        if ns.accept_labels:
+            self.accept_labels: tuple[str, ...] = tuple(ns.accept_labels)
```

Comment:
Maybe `set[str]`?

Reply:
For some reasons they should be immutable, as well as `match_tests` and `ignore_tests`.

In any case, I am planning to rewrite filtering by names. Instead of two tuples there will be more complex structure where order matters. And the same can be used for labels.